### DISCUSSION TITLE
22.4.44

### DISCUSSION THREAD

**Discussion by immauss** [immauss/openvas#268] [Announcements]
Minor update from Greenbone...

And .... 

The addition of .notus files for Rocky Linux. 

Currently, the image is configured to pull the .notus feed from rsync.immauss.com and all other feeds are from the greenbone community feed servers. The Immauss rsync feed is updated daily with the GB community feed an the Rocky Linux feed will be update every other month. For more info, read [here](https://www.immauss.com/gb-feeds).

Thanks,
Scott

<hr /><em>This discussion was created from the release <a href='https://github.com/immauss/openvas/releases/tag/'>22.4.44</a>.</em>

**Comment by kjake** [comment]
FYI - the site you linked has a 404 and untrusted cert (https://www.immauss.com/gb-feeds) for me.

**Comment by immauss** [comment]
Strangely ... all I did was restart httpd .... 

Should be back up now. Thanks for the note.

I rebooted it earlier today after updates.  Looks like it had trouble knowing who it was went it started the first time ..... odd indeed. 

Thanks again @kjake 

-Scott
